components/form.js

```diff
@@ -1052,5 +1054,43 @@ function Client (Component) {
   }
 }
 
+function PasswordHider ({ onClick, showPass }) {
+  const handleClick = () => {
+    onClick()
+  }
```

Comment:
Looks like an artifact and no longer needed. You can simply pass `onClick` to `onClick`, no?